Countries Info/Helpers/PaddingLabel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PaddingLabel.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 11.12.2020.
+//
+
+import UIKit
+
+class PaddingLabel: UILabel {
+
+    var topInset: CGFloat
```

Comment:
We can have only one property here `UIEdgeInsets `, or even pass it as a param in initialiser